src/job.ts

```diff
@@ -379,6 +381,11 @@ class Job extends Operation {
     );
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  private trace_(msg: string, ...otherArgs: any[]) {
```

Comment:
Is this temporary functionality, or do we do much more extensive logging in node?  This feels like it would be better done via otel or similar?